src/graphql/schema/createGqlSchema.ts

```diff
@@ -40,6 +43,6 @@ export default function createGqlSchema (inventory: Inventory, options: SchemaOp
 
   return new GraphQLSchema({
     query: getQueryGqlType(buildToken),
-    mutation: getMutationGqlType(buildToken),
+    mutation: getMutationGqlType(buildToken, options.disableDefaultMutations),
```

Comment:
All of the options, instead of being passed directly go into the [`buildToken`](https://github.com/calebmer/postgraphql/pull/180/files#diff-094e28372a57698d7f184ae8cadf05f6R34) object. This ensures that we can add a default and that the options are available anywhere while we build our GraphQL objects (as the `buildToken` is passed into everything).